### PR TITLE
completed javascript-if

### PR DESCRIPTION
**Pull request by BrianBozi**
image

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 21) {
```
Comment (scbowler) [inline javascript-if/main.js]: Should be checking if under 5 not 21

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+function categorizeAcidity(pH) {
+  if (pH < 7) {
+    return 'acid';
+  } if (pH === 7) {
+    return 'nuetral';
+  } if (pH < 14) {
+    return 'base';
+  } else {
+    return 'invalid pH level';
+  }
```
Comment (scbowler) [inline javascript-if/main.js]: If `ph` is `-20` this will return `acid` when it should return `invalid ph level`. Needs a little reworking on the logic.

**Review by scbowler — CHANGES_REQUESTED** [review]
A couple small changes needed

**Comment by BrianBozi** [comment]
image
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**